src/js/view/controls/shortcuts-tooltip.js

```diff
@@ -80,8 +81,9 @@ export default function (container, api, model) {
         isOpen = true;
         api.pause(settingsInteraction);
     };
+
     const close = () => {
-        shortcutToggle.removeEventListener('click', toggleClickHandler);
+        shortcutToggleUi.off();
```

Comment:
Why do we remove the listeners (ui.off) when it closes, and add them again when it opens? If there isn't a specific reason, we could add once during setup (in render method) and destroy when component is destroyed. If git lens says

Reply:
Good call! No reason I know of.